R/bmerge.R

```diff
@@ -75,12 +75,12 @@ bmerge = function(i, x, icols, xcols, roll, rollends, nomatch, mult, ops, verbos
     if (xclass=="character" || iclass=="character" ||
         xclass=="logical" || iclass=="logical" ||
         xclass=="factor" || iclass=="factor") {
-      if (anyNA(i[[ic]]) && all(is.na(i[[ic]]))) { # TODO: allNA function in C
+      if (anyNA(i[[ic]]) && allNA(i[[ic]])) {
```

Comment:
this change is nice but I would hold that for now, we support only few classes in `allNA`, are we sure none of the others can reach here.

Reply:
good idea.  would agree but I'm not sure what we'd wait for: all tests and all revdeps pass.  So might as well make progress.  Plus is quite a simple function:  hard to get wrong (touch wood!)

Reply:
@mattdowle I pushed that in extra commit now, could you please re-run revdeps? it was previously returning `false` by default, so the potential issues could have been missed. Now it is more explicit and will raise error.